src/features/theme-switcher/ui/ThemeSwitcher.css

```diff
@@ -0,0 +1,7 @@
+.Space {
```

Comment:
Is this consistent with the AntD documentation?

Reply:
Ant design for styling space use inline styles, we agreed to use className instead of inline styles.
Example from ant-design: 
![image](https://user-images.githubusercontent.com/48411208/206223182-eddc1e3f-fe64-471e-baf6-b4dbb08403b7.png)